source/functions/BEMIO/writeBEMIOH5.m

```diff
@@ -67,7 +67,7 @@ function writeBEMIOH5(hydro)
         writeH5Parameter(filename,['/body' num2str(i) '/hydro_coeffs/linear_restoring_stiffness'],tmp(1,m_add + 1:m_add + m,:),'Hydrostatic stiffness','N/m');
         clear tmp;
     else
-        writeH5Parameter(filename,['/body' num2str(i) '/hydro_coeffs/linear_restoring_stiffness'],hydro.Khs(:,:,i),'Hydrostatic stiffness matrix','');        
+        writeH5Parameter(filename,['/body' num2str(i) '/hydro_coeffs/linear_restoring_stiffness'],hydro.Khs(:,:,i)','Hydrostatic stiffness matrix','');        
```

Comment:
@salhus I do not have an .h5 viewer, but you mention that this is flip is to make the viewer have the right matrix format; however, we then use the reverseDimensionOrder in the readBEMIOH5 to get it back in the original form for WEC-Sim. Therefore, we are flipping it forward and flipping it backward. I tried running without the transpose and then removing the reverseDimensionOrder and the WAMIT and AQWA matrices were in the right format for WEC-Sim. 

Therefore, my question is if we can clean up this implementation to not have unnecessary flipping if it is only for the .h5 viewer. As mentioned above the results come out as we want, just seeing if the implementation can be modified.

Reply:
@nathanmtom,
I see your point that it is redundant. However, I think having the h5 file represent the coefficients is also important.
The h5 files are useful in their own right because it can be a standardised way of expressing the bemio processed results regardless of BEM software being used. This is helpful when comparing hydrodynamic coefficients, or importing it to optimizers etc.

Reply:
@salhus I understand and agree with that point. But now wondering why the .h5 viewer does a flip when importing into its own data format. Did you have the opportunity to verify if other .h5 viewers provided the same result? As perhaps if there is an import issue with the one .h5 viewer you are testing while others do not have this issue then perhaps we eliminate this redundancy and note one .h5 viewer is perhaps importing incorrectly. This might be a non-issue just do not personal use .h5 viewers at this time.

Reply:
@nathanmtom 
Yes, I did check with another h5viewer, and observed the same dimension order.